.circleci/config.yml

```diff
@@ -162,7 +162,18 @@ jobs:
             cf api https://api.fr.cloud.gov
             cf auth "$CF_USERNAME" "$CF_PASSWORD"
             cf target -o $CF_ORG -s "$CF_SPACE"
-            ./deploy-cloudgov-docker.sh rolling test $CGHOSTNAME_BACKEND $CGHOSTNAME_FRONTEND $DOCKER_IMAGE_BACKEND $DOCKER_IMAGE_FRONTEND $CIRCLE_BRANCH
+
+            # Deploy ClamAV REST server
+            # First delete the existing implementation of clamav-rest so we don't exhaust memory quota
+            cf delete -r clamav-rest -f
```

Comment:
We can't use a rolling deployment strategy for this one since it would require double resource usage during the deployment and our space quota does not allow for this. Instead, we'll delete the existing app first and then re-deploy.

Long term it would be good to conditionally skip re-deploying if nothing has changed with clamav-rest and it is already running.